helix-term/src/compositor.rs

```diff
@@ -76,9 +89,14 @@ pub struct Compositor {
 }
 
 impl Compositor {
-    pub fn new() -> Result<Self, Error> {
+    pub fn new() -> anyhow::Result<Self> {
+        #[cfg(not(feature = "integration"))]
         let backend = CrosstermBackend::new(stdout());
-        let terminal = Terminal::new(backend)?;
+
+        #[cfg(feature = "integration")]
+        let backend = TestBackend::new(120, 150);
```

Comment:
I wish we could also test out `80, 24` or at least allow testing it with a smaller terminal.

Reply:
This is not a real terminal instance, it's a facade. I'm not sure if the size matters at all.

Reply:
Size matters for display I believe, like to test out whether or not we display CJK characters correctly, but I think the current test does not include testing everything within screen but instead test what was the output document. Later going forward we probably want to have something to test out of the display is correct.

Reply:
Hmm, okay I don't really have a good idea about that, as I don't have any experience with that code. Is this something that could be improved when those tests are written?

Reply:
I am not very sure as well, but I don't recall kakoune testing UI content, it only test the document content.